e2e/LongHaul/device/Program.cs

```diff
@@ -16,23 +14,30 @@ namespace ThiefDevice
     internal class Program
     {
         private static readonly IDictionary<string, string> s_commonProperties = new Dictionary<string, string>();
-        private static Settings s_settings;
         private static Logger s_logger;
         private static ApplicationInsightsLoggingProvider s_aiLoggingProvider;
+        private static readonly string s_deviceConnectionString = TestConfiguration.DeviceConnectionString;
```

Comment:
Nope, should be created by the ARM template, saved as a key vault secret, and fetched as an env var.